webapp/commands.py

```diff
@@ -77,6 +77,24 @@ def import_assets_from_prod(token):
         if entry.get("optimized", None):
             entry["optimize"] = entry.get("optimized", None)
 
+        image_extensions = [
+            "jpeg",
+            "jpg",
+            "gif",
+            "png",
+            "apng",
```

Comment:
Never seen this before

Reply:
In the assets server when someone uploads an asset we verify the asset MIME type by the magic number at the beginning of the file, in this scenario that is very expensive, as we will have to download around 20 000 assets, so it seemed more reasonable to check the assets file path extension especially that the assets comes from a place where there is already a magic number check already done.